docs/quickstart/setup.rst

```diff
@@ -50,6 +54,63 @@ networks can overlap, and nodes need not have distinct IPs for each plane.
 In order to reach the cluster-provided UIs from your host, the host needs to be
 able to connect to workload-plane IPs of the machines.
 
+Repositories provisioning
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Each machine needs to have repositories properly configured and access to basic
+repositories packages (depending on the operating systems).
```

Comment:
Each machine needs to have repositories properly configured and having access to basic repository packages( depending on the operating systems). ??